lib/active_shipping/carriers/usps.rb

```diff
@@ -168,15 +168,15 @@ class USPS < Carrier
       /Delivery status information is not available/
     ]
 
-    ESCAPING_AND_SYMBOLS = /&amp;lt;\S*&amp;gt;/
-    LEADING_USPS = /^USPS/
+    ESCAPING_AND_SYMBOLS = /&lt;\S*&gt;/
+    LEADING_USPS = /^USPS /
     TRAILING_ASTERISKS = /\*+$/
     SERVICE_NAME_SUBSTITUTIONS = /#{ESCAPING_AND_SYMBOLS}|#{LEADING_USPS}|#{TRAILING_ASTERISKS}/
 
     def find_tracking_info(tracking_number, options = {})
       options = @options.update(options)
       tracking_request = build_tracking_request(tracking_number, options)
-      response = commit(:track, tracking_request, (options[:test] || false))
+      response = commit(:track, tracking_request, options[:test] || false)
```

Comment:
What is `options[:test] || false` trying to accomplish? Coercing to a true/false class? Because it will only do so for the false case but the true case will pass through. Or to simplify, this means that every value will be passed through without changing, except for `nil` which will be turned into `false`.